codegenerator/cli/npm/envio/src/db/Table.res

```diff
@@ -240,11 +240,17 @@ module PostgresInterop = {
     Promise.all(responses)
   }
 
-  let makeBatchSetFn = (~table, ~rowsSchema: S.t<array<'a>>): batchSetFn<'a> => {
+  let makeBatchSetFn = (~table, ~schema: S.t<'a>): batchSetFn<'a> => {
     let batchSetFn: pgFn<array<Js.Json.t>, unit> = table->makeBatchSetFnString->eval
+    let parseOrThrow = S.compile(
+      S.array(schema),
+      ~input=Value,
+      ~output=Json,
+      ~mode=Sync,
+      ~typeValidation=true,
+    )
```

Comment:
Going to need to some explanations on these new features

Reply:
Here's the doc page https://github.com/DZakh/rescript-schema/blob/main/docs/rescript-usage.md#compile